### PR TITLE
fix(ui): manage timezone in details panel

### PR DESCRIPTION
**Pull request by kduret**
## Description

Manage user timezone in events view details panel

**Fixes** MON-5317

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

* Update user timezone
* Check host / service details display date in the proper timezone

### CODE REVIEW

File: www/front_src/src/Resources/dateTime.ts [hunk 1]
```diff
@@ -1,14 +1,13 @@
-import format from 'date-fns/format';
-import parseISO from 'date-fns/parseISO';
+import moment from 'moment';
```
Comment (bdauria) [inline www/front_src/src/Resources/dateTime.ts:1]: We already embed full moment in the bundle right?

Comment (kduret) [reply www/front_src/src/Resources/dateTime.ts:1]: Yes, and partially moment-timezone

**Comment by Thebarda** [comment]
aaa
It seems to work when opening Downtime dialog

dsdsd
After updating check, it does not update last check date time in the correct timezone

**Comment by Thebarda** [comment]
Tested with last build and it works perfectly

**Pull request merged**